sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -97,6 +101,7 @@ public final class CosmosAsyncClient implements Closeable {
             ImplementationBridgeHelpers.CosmosContainerIdentityHelper.getCosmosContainerIdentityAccessor();
     private final ConsistencyLevel accountConsistencyLevel;
     private final WriteRetryPolicy nonIdempotentWriteRetryPolicy;
+    private final ProactiveOpenConnectionsProcessor proactiveOpenConnectionsProcessor;
```

Comment:
`RxDocumentClientImpl` maybe a better place to initialize `ProactiveOpenConnectionsProcessor`

Reply:
Fixed